dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -445,17 +445,17 @@ Sedp::init(const RepoId& guid,
 #endif
 
   if (spdp_.available_builtin_endpoints() & DISC_BUILTIN_ENDPOINT_PARTICIPANT_ANNOUNCER) {
```

Comment:
```suggestion
  if (spdp_.available_builtin_endpoints() & BUILTIN_ENDPOINT_PARTICIPANT_MESSAGE_DATA_WRITER) {
```
Need to review use of these constants.